extension/tests/onboarding.test.ts

```diff
@@ -0,0 +1,144 @@
+import { shuffle } from "lodash";
+import { test, expect } from "./test-fixtures";
+
+test.beforeEach(async ({ page, extensionId }) => {
+  await page.goto(`chrome-extension://${extensionId}/index.html`);
+});
+
+test("Welcome page loads", async ({ page }) => {
+  await expect(
+    page.getByText("Welcome! Is this your first time using Freighter?"),
+  ).toBeVisible();
+  await expect(page.getByText("I’m going to need a seed phrase")).toBeVisible();
+  await expect(page.getByText("I’ve done this before")).toBeVisible();
+});
+
+test.only("Create new wallet", async ({ page }) => {
```

Comment:
let's omit `.only`

Reply:
ah whoops, thanks.

Reply:
removed in e99194053f8fb93000911ce5cb43589f3ef43894